### PR TITLE
[RA2 Ch02 - Ch03] Add support for package management APIs

### PR DESCRIPTION
**Pull request by xmulligan**
#946 adding support for package management as a requirement

### CODE REVIEW

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 1]
```diff
@@ -235,7 +235,6 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | `req.kcm.01` | General | The Architecture **must** allow infrastructure resource sharing within a Kubernetes cluster. |
 | `req.kcm.02` | General | The Architecture **must** support discoverability of nodes and their features. |
 | `req.kcm.03` | General | The Architecture **must** support scheduling of workloads based on Enhanced Platform Awareness (EPA) features such as CPU Pinning, huge-pages and SR-IOV. |
-| `req.kcm.04` | General | The Architecture **must** include kubernetes artefacts (e.g., images, Helm charts, etc.) repository capabilities. |
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: I'm not sure why this line has been deleted?

File: doc/ref_arch/kubernetes/chapters/chapter02.md [hunk 2]
```diff
@@ -249,6 +248,7 @@ Please note that "shared" is a reference to multi-tenant support and pooled stor
 | Ref # | sub-category | Description |
 |---|---|---|
 | `req.int.api.02` | API | The Architecture **must** leverage the Kubernetes APIs to discover and declaratively manage compute (NFVI and bare metal resources), network, and storage. |
+| `req.kcm.04` | API | The Architecture **must** support the usage of a Kubernetes Application package manager using the Kubernetes API-s, like Helm v3. |
```
Comment (tomkivlin) [inline doc/ref_arch/kubernetes/chapters/chapter02.md]: Can the reference be, e.g. `req.int.api.02` in line with the rest of the table?

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 1 above)
Comment (xmulligan) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: You're right didn't mean to delete this. Did ctr-x not ctr-c. On a related note, should this also be moved as a requirement under API. Something like "The Architecture **must** support the usage of an OCI compatible artefact repository.

Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: Yes agreed- could you raise an Issue for that please?

Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: And could you add this back in please?

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 2 above)
Comment (tomkivlin) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: Ah sorry that should be `req.int.api.03` as the one above is already `02`.

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 1 above)
Comment (xmulligan) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: added in #1108 and will add back in

File: doc/ref_arch/kubernetes/chapters/chapter02.md (hunk 2 above)
Comment (xmulligan) [reply doc/ref_arch/kubernetes/chapters/chapter02.md]: fixed

**Pull request merged**